### PR TITLE
✨ Create History Page

### PR DESCRIPTION
**Pull request by mrana17**
- Header
- SubHeader
- Order Cards
- Footer

### CODE REVIEW

File: pages/HistoryPage.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import HeaderButton from "../components/headerButton/headerButton";
+import SubHeader from "../components/subHeader/subHeader";
+import styles from "../styles/HistoryPage.module.css";
+import "react-calendar/dist/Calendar.css";
```
Comment (mewiemers) [inline pages/HistoryPage.tsx]:
```suggestion

```

File: pages/HistoryPage.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          onClick={() => alert("Notifications will follow")}
+        />
+        <SubHeader
+          className={styles.subHeader}
+          infoOne="Fahrten"
+          infoTwo="Verdienst"
+          infoThree="Trinkgeld"
+          infoOneNumber={16}
+          infoTwoNumber={[50.5, "€"]}
+          infoThreeNumber={[25.1, "€"]}
```
Comment (mewiemers) [inline pages/HistoryPage.tsx:23]: could you name that more descriptive?

File: pages/HistoryPage.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+//       <div className={styles.cardContainer}>
+//         <OrderCard
+
+//         />
+//       </div>
+//     </main>
+//   );
+// }
```
Comment (mewiemers) [inline pages/HistoryPage.tsx]: do you need this anymore?

**Review by mewiemers — APPROVED** [review]
great job, just a few comments 🛺

File: pages/HistoryPage.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+          className={styles.moreButton}
+          imgSrc="/more.svg"
+          onClick={() => alert("more functions will follow")}
+        />
+      </div>
+      <div className={styles.cardContainer}>
+        <OrderCard
+          imgSrc="/Card-Placeholder.png"
+          imgSrcStart="/Start.svg"
+          imgSrcEnd="/Target.svg"
```
Comment (phlgr) [inline pages/HistoryPage.tsx:35]: Isn't the start and end Image always the same? If so you could just set it in the component

**Pull request merged**